packages/react-components/src/components/VideoGallery.tsx

```diff
@@ -396,6 +396,10 @@ export interface VideoGalleryProps {
    * Controls the view of the local screenshare stream in the gallery
    */
   localScreenShareView?: LocalScreenShareView;
+  /**
+   * Boolean to indicate if it is mobile or desktop
+   */
+  isMobileView?: boolean;
```

Comment:
don't add `isMobile` to components. Component props should be specific and explicit about purpose, isMobile is a catch-all property that doesn't tell the developer what will happen to the component when applied, it also will impact multiple aspects. Instead have something like `showNameplate` and in the Composite layer do `<VideoGallery showNameplate={isMobile} />`

Reply:
What about showMobileVIew or showCompactView? Based on our discussion previously, we want to only show the video gallery compact view based on isMobile instead of determine based on width. https://github.com/Azure/communication-ui-library/pull/5643

Reply:
What precisely does compact view do? How many things change?

Reply:
For now it's showing label on local video tile. In the future we might have other UI added for mobile video gallery, we want to start using the isMobile value to make sure when on dekstop we are forcing the videoGallery to not trip is narrow and to always be the desktop mode

Reply:
For those few items we should have individual properties for. What if a developer wants a narrow experience on desktop? Or for ipads/tablets that have larger screens are they isMobile? (They may want some ismobile properties like larger touch targets, or swipe-able overflow gallery, but not other ismobile properties that save space like hiding display name). When we bucket many different properties under a single property we remove flexibility for developers and only make our component better for the one experience we have in mind today.

Reply:
I started a group chat with you and Don to discuss this API further.